sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/arm-templates/cluster.parameters.json

```
@@ -17,6 +17,9 @@
     "certificateThumbprint": {
         "value": "<REPLACE THIS: Hexadecimal string that can be found as Thumbprint or X.509 SHA-1 Thumbprint (in hex) in the certificate's properties>"
     },
+    "sshKeyData": {
+        "value": "<REPLACE THIS: If providing an SSH key, this should be the base64-encoded SSH public key (the content of the key's .pub file, with no trailing whitespace). Looks like ssh-rsa {base64 content} {user}@{machine}>"
```

It must be provided, and is it only the base64 encoded content or an SSH public key one-liner?
```suggestion
        "value": "<REPLACE THIS: an SSH public key (the content of the key's .pub file, with no trailing whitespace). Looks like ssh-rsa {base64 content} {user}@{machine}>"
```

---

Good point -- "base64-encoded SSH public key" came from ARM's documentation, but it should but the full public key. Thanks!